packages/core/admin/ee/admin/content-manager/pages/EditView/InformationBox/InformationBoxEE.js

```diff
@@ -14,8 +14,10 @@ import { useReviewWorkflows } from '../../../../pages/SettingsPage/pages/ReviewW
 import { OptionColor } from '../../../../pages/SettingsPage/pages/ReviewWorkflows/components/Stages/Stage/components/OptionColor';
 import { SingleValueColor } from '../../../../pages/SettingsPage/pages/ReviewWorkflows/components/Stages/Stage/components/SingleValueColor';
 import Information from '../../../../../../admin/src/content-manager/pages/EditView/Information';
+import { useAdminUsers } from '../../../../../../admin/src/hooks/useAdminUsers';
 
-const ATTRIBUTE_NAME = 'strapi_reviewWorkflows_stage';
+const STAGE_ATTRIBUTE_NAME = 'strapi_reviewWorkflows_stage';
+const ASSIGNEE_ATTRIBUTE_NAME = 'strapi_reviewWorkflows_assignee';
```

Comment:
Are these attribute names still correct? I thought we were going for `strapi_assignee`? Or do you plan to update that later ?

Reply:
No good point - I'll keep the one for stage, but update the one for assignee 👍🏼

Reply:
Updated & rebased.